visionipc/tests/test_visionipc.py

```diff
@@ -0,0 +1,71 @@
+import os
+import gc
+import time
+import random
+import unittest
+import numpy as np
+from cereal.visionipc import VisionIpcServer, VisionIpcClient, VisionStreamType
+
+def zmq_sleep(t=1):
+  if "ZMQ" in os.environ:
+    time.sleep(t)
+
+
+class TestVisionIpc(unittest.TestCase):
+
+  def tearDown(self):
+    if hasattr(self, 'server'):
+      del self.server
+      gc.collect()
```

Comment:
why is this necessary?

Reply:
I was seeing occasional `Error, failed to bind PubSocket to 9000: Address already in use` errors in CI, wanted to check if this fixed it